packages/wonder-blocks-form/src/components/field-heading.js

```diff
@@ -121,14 +121,15 @@ export default class FieldHeading extends React.Component<Props> {
     maybeRenderError(): ?React.Node {
         const {error, id, testId} = this.props;
 
-        if (!error) {
-            return null;
-        }
-
+        // NOTE(jeresig, FEI-3472): We need to always display the error
+        // element, even if there is no error. This is because the error
+        // uses an ARIA alert role, which requires that the element be
+        // in the DOM before the text is inserted. Doing otherwise will
+        // mean that the alert will not be read.
         return (
             <React.Fragment>
-                <Strut size={Spacing.small_12} />
-                {typeof error === "string" ? (
+                {error && <Strut size={Spacing.small_12} />}
+                {typeof error === "string" || !error ? (
```

Comment:
Super nit: I was taking a look at the source code, and it looks that we can even simplify this more.

We currently define error as: `error: string | React.Element<Typography>`, but...

we are only passing `error: string` from the callers, so maybe we can simplify the `error` type:

```
error?: string,
```

That way we can move the condition inside line # 139.

Of course no need to do it here, but wanted to point out that finding :)